CONTRIBUTING.rst

```diff
@@ -162,6 +162,59 @@ Contributors are responsible for:
 * Adding features
 * Championing one or more items on the `Roadmap <https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Home>`__.
 
+Security Team
+-------------
+
+Security issues in Airflow are handled by the Airflow Security Team. The team consists
```

Comment:
This is mostly unchanged, but I did reword a few trivial things in here (e.g. dropped "The" from the first sentence).